layout/layout.ejs

```diff
@@ -27,7 +27,10 @@
   }
   %>
   <title><%= title %></title>
-
+  <% if (config.sitemap.rel) { %>
```

Comment:
Here `config.sitemap.rel` is undefined, is that referring to https://github.com/hexojs/hexo-generator-sitemap?

Reply:
Yes it's preparation for this plugin, if somebody wants to use it, theme will be ready for it.
I use it on my site ;)

Reply:
Also it is weird that it doesn't work as expected, it should simply ignore this entry if there is no sitemap in config, but it just ignores the if check and tries it anyway, not sure why.. im checking it